src/pykeen/models/predict.py

```diff
@@ -1,25 +1,164 @@
 # -*- coding: utf-8 -*-
 
-"""Prediction workflows."""
+"""
+Prediction workflows.
 
+This module contains methods to score triples or prediction targets with a given model.
+For the remainder of this part of the documentation, we assume that we have trained a model, e.g. via
+
+>>> from pykeen.pipeline import pipeline
+>>> result = pipeline(dataset="nations", model="pairre", training_kwargs=dict(num_epochs=0))
+
+
+High-Level
+==========
+The prediction workflow offers three high-level methods to perform predictions
+
+- :func:`pykeen.models.predict.predict_triples` can be used to calculate scores for a given set of triples.
+- :func:`pykeen.models.predict.predict_target` can be used to score choices for a given prediction target, i.e.
+  calculate scores for head entities, relations, or tail entities given the other two.
+- :func:`pykeen.models.predict.predict` can be used to calculate scores for all possible triples.
```

Comment:
maybe better to call this `predict_all()`?

Reply:
https://github.com/pykeen/pykeen/pull/1090/commits/6adcee58ab61f4be7466f83ffb177f1c424124aa